doc/_themes/sphinx13/static/sphinx13.css

```diff
@@ -218,6 +218,20 @@ div.sphinxsidebar #searchbox {
     margin: 12px 0 20px 0;
 }
 
+div.sphinxsidebar #searchbox input[type="text"] {
+    border-radius: var(--admonition-radius) 0 0 var(--admonition-radius);
+}
+
+div.sphinxsidebar #searchbox input[type="text"] {
+    border-radius: var(--admonition-radius) 0 0 var(--admonition-radius);
+}
```

Comment:
I was about to merge this, but these two rules are identical -- is this intentional?

Reply:
Sorry, this was not intentional. I've removed the duplicate code.